docs/source/plugins/developing_plugins.rst

```diff
@@ -1885,7 +1891,7 @@ subsequent sections.
             ctx.panel.set_data("event_data", event)
 
             # Get the list of brain keys to populate `brain_key` dropdown
-            visualization_keys = ctx.dataset.list_brain_runs("visualization")
```

Comment:
We should probably keep this filter. Non-visualization brain runs do not have points in one of on_change method used for the dropdown.

Reply:
Merged. My mistake. I have a follow up you think we should revert https://github.com/voxel51/fiftyone/pull/4737